hetnet/neo4j/deployment/setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# This shell script deploys neo4j-hetionet web server on Ubuntu 18.04.
+
+# Run the script as user "ubuntu" only.
+if [ `whoami` != 'ubuntu' ]; then
+    echo "Error: only the user 'ubuntu' is allowed to run this script."
+    exit 1
+fi
+
+# Ensure that the working directory is correct
+cd `dirname $0`
+
+# Update packages automatically using a daily cron job
+sudo apt update
+sudo apt purge unattended-upgrades --yes
+sudo rm -rf /var/log/unattended-upgrades/
+sudo cp upgrade-pkg /etc/cron.daily/
+sudo chmod 755 /etc/cron.daily/upgrade-pkg
+
+# "root" bash config
+sudo cp -f bash_aliases /root/.bash_aliases
+sudo bash -c "cat root_bash_prompt >> /root/.bashrc"
+
+# "ubuntu" bash config
+cp -f bash_aliases ~/.bash_aliases
+
+# Customize hostname
+sudo bash -c "echo neo4j-hetionet > /etc/hostname"
+
+# Install the latest Docker CE (Community Edition)
+sudo apt install apt-transport-https gnupg-agent --yes
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
long args for readability? We should use long args everywhere they are supported for all scripts we track with git.